.github/workflows/create-py-release-manylinux.yaml

```
@@ -1,20 +1,41 @@
 name: Create Python release manylinux
 
 on:
-  push:
-    # Sequence of patterns matched against refs/tags
-    tags:
-      - 'py-polars-v*' # Push events to matching py-polars-v*, i.e. py-polars-v1.0, py-polars-v20.15.10
+  - pull_request
+#  push:
+#    # Sequence of patterns matched against refs/tags
+#    tags:
+#      - 'py-polars-v*' # Push events to matching py-polars-v*, i.e. py-polars-v1.0, py-polars-v20.15.10
 jobs:
   build_manylinux:
     name: Create Release manylinux
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Publish wheel
-        uses: docker://konstin2/maturin:latest
+#      - name: Publish wheel
+#        uses: docker://konstin2/maturin:latest
+#        env:
+#          MATURIN_PASSWORD: ${{ secrets.PYPI_PASS }}
+#        with:
+#          entrypoint: /bin/bash
+#          args: .github/deploy_manylinux.sh
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Prepare maturin publish
+        shell: bash
+        run: |
+          rm py-polars/README.md
+          cp README.md py-polars/README.md
+      - name: maturin publish
+        uses: messense/maturin-action@main
         env:
           MATURIN_PASSWORD: ${{ secrets.PYPI_PASS }}
         with:
-          entrypoint: /bin/bash
-          args: .github/deploy_manylinux.sh
+          rust-toolchain: nightly-2022-01-01
```

Too bad that the manylinux2014 aarch64 docker image only has GCC 4.8.5 which misses `stdatomic.h`, maybe we can try manylinux_2_24 or disable mimalloc for it.

```suggestion
          rust-toolchain: nightly-2022-01-01
          manylinux: 2_24
```

---

Ok, another shot!